src/OtpInput.tsx

```diff
@@ -63,11 +65,14 @@ export default class OtpInput extends Component<Props, State> {
       value,
     } = this.props
 
+    console.log(focusStyles)
```

Comment:
Remove logs :)

Reply:
oops, my bad, i thought i discarded this one ;) Done !